control-plane/subcommand/inject-connect/command.go

```diff
@@ -697,11 +697,9 @@ func (c *Command) Help() string {
 	return c.help
 }
 
-const synopsis = "Inject Connect proxy sidecar."
+const synopsis = "Inject the Consul service mesh dataplane."
```

Comment:
I think this current description is more generic and easier to understand, and so it doesn't matter if we're injecting consul-dataplane or something else since it's just a sidecar proxy. The only thing I'd change is the word "connect."

Also, since we're updating it, we should mention the full functionality of this command: that it runs the endpoint controller and the peering controllers as well (not just injecting the sidecar)

Reply:
Agreed. Let me know what you think of the new wording.